mmv1/third_party/validator/tests/data/example_project_create.tfplan.json

```diff
@@ -52,7 +52,8 @@
         "before": null,
         "after": {
           "display_name": "Department 1",
-          "parent": "organizations/1234567",
+          "folder_id": "{{.FolderID}}",
```

Comment:
folder_id is an output-only field; it doesn't need to be added here if it wasn't in the plan previously.

```suggestion
```